tests/dm_control/test_dm_control_envs.py

```diff
@@ -0,0 +1,32 @@
+import pickle
+import unittest
+
+from dm_control.suite import ALL_TASKS
+import numpy as np
+
+from garage.envs.dm_control import DmControlEnv
+
+
+class TestDmControlEnvs(unittest.TestCase):
```

Comment:
A better way to construct this test is as follows:

Make the TestCase (not the method) parameterized. In setup, create the environment. Then add test methods for vanilla stepping `test_can_step_and_render()` and pickling `test_pickling()`.

This way, if the pickling test fails, the error will point right to the pickling test, not into the middle of a large test.